packages/i18n/src/locales/en.i18n.json

```diff
@@ -4234,7 +4234,7 @@
   "Please_fill_all_the_information": "Please fill all the information",
   "Please_fill_an_email": "Please fill an email",
   "Please_fill_name_and_email": "Please fill name and email",
-  "Please_fill_out_reason_for_report": "Please fill out the reason for the report",
+  "Please_fill_out_reason_for_report": "Reason for report required",
```

Comment:
Looking at `apps/meteor/client/views/room/contextualBar/UserInfo/ReportUserModal.tsx`, it feels like this string only exists because of the improper design of the form. The field label in question visually is a user name, but in terms of a11y it should be “Reason for report” or “Description“. We can have both, but the former should be decorative (i.e. marked with `role="presentation"`). This would enable us to replace `register('description', { required: t('Please_fill_out_reason_for_report') })` with `register('description', { required: t('Required_field', { field: t('Reason_for_report') }) })`.